packages/react-experiments/src/components/CollapsibleSection/CollapsibleSection.state.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { useCallback, useImperativeHandle, useRef } from 'react';
```

Comment:
do we use this? I would be very cautious about modifying code that we're not the ones actively developing